reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md

```diff
@@ -286,10 +307,11 @@ provider.
 
 ### .FORWARDHELPCATEGORY
 
-Specifies the help category of the item in "ForwardHelpTargetName". Valid
-values are "Alias", "Cmdlet", "HelpFile", "Function", "Provider", "General",
-"FAQ", "Glossary", "ScriptCommand", "ExternalScript", "Filter", or "All". Use
-this keyword to avoid conflicts when there are commands with the same name.
+Specifies the help category of the item in `.ForwardHelpTargetName`. Valid
+values are **Alias**, **Cmdlet**, **HelpFile**, **Function**, **Provider**,
```

Comment:
per our other conversation about valid values, these should be code fenced

Reply:
Fixed